openreview/venue/invitation.py

```diff
@@ -3095,4 +3095,132 @@ def set_SAC_ethics_flag_invitation(self, sac_ethics_flag_duedate=None):
             if sac_ethics_flag_duedate:
                 invitation.edit['invitation']['expdate'] = tools.datetime_millis(sac_ethics_flag_duedate)
 
-            self.save_invitation(invitation, replacement=True)
\ No newline at end of file
+            self.save_invitation(invitation, replacement=True)
+
+    def set_group_recruitment_invitations(self, committee_name):
+
+        venue_id = self.venue_id
+        venue = self.venue
+
+        invitation = Invitation(id=venue.get_group_recruitment_id(committee_name),
+            invitees=[venue_id],
+            readers=[venue_id],
+            writers=[venue_id],
+            signatures=['~Super_User1'],
```

Comment:
@melisabok I remember we spoke about using Super User for now, and that using the venue id should work too like you are using it for the journal.

If I am remebering correctly, `memberships = [g.id for g in client.get_groups(member=email, prefix=venue_id)]` would give an error only if the `email` group does not exist in OR. However, I am already checking this:
`memberships = [g.id for g in client.get_groups(member=email, prefix=venue_id)] if openreview.tools.get_group(client, email) else []` and I am still getting the following error with a profile ID that exists: 
`User Venue is not reader of ~Reviewer_VenueThree1`